docs.md

```diff
@@ -192,6 +192,11 @@ The system project cannot be deleted.
 
 Project quotas and default limits must be consistent with one another and must be sufficient for the requirements of active namespaces.
 
+#### Container default resource limit validation
+
+Container default resource limit configuration must have no negative values for any requests and limits.
```

Comment:
Nit: I would recommend trying to indicate that these limitations are sourced from upstream in some way. While you don't have to link to the docs, you could try and say something like "The container default limit configuration must have no negative values for any requests and limits, and the values must be able to be placed into a valid LimitRange". That way we are clear that we are mirroring limitations from upstream, rather than inventing our own.

Reply:
Agreed!